lib/modules/datasource/bazel-registry/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`modules/datasource/bazel-registry/index getReleases metadata with yanked versions 1`] = `
+{
+  "registryUrl": "https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main/modules/rules_foo/metadata.json",
+  "releases": [
+    {
+      "registryUrl": "https://raw.githubusercontent.com/bazelbuild/bazel-central-registry/main",
```

Comment:
This looks like it should be the same `registryUrl`

Reply:
Perhaps, it was unclear to me whether the URL was meant to be the original registry URL or the fully resolved location of the registry information for the module.

Reply:
Per your other comment, I am removing it.